packages/serverless-logic-web-tools/src/i18n/locales/de.ts

```diff
@@ -64,6 +64,9 @@ export const de: AppI18n = {
       hostInfo: `Der mit Ihrer Instanz verbundene Hostname.`,
       namespaceInfo: `Der Namespace (Projekt), in dem Sie das Modell bereitstellen möchten.`,
       tokenInfo: `Das mit Ihrer Instanz verknüpfte Token.`,
+      insecurelyDisableTlsCertificateValidation: "Unsichere Deaktivierung der TLS Zertifikat Validierung.",
```

Comment:
```suggestion
      insecurelyDisableTlsCertificateValidation: "Unsichere Deaktivierung der TLS Zertifikat Validierung",
```
Removing the final period.